### PR TITLE
mLab Online Database

### PR DESCRIPTION
**Pull request by kevinpr007**
# mLab Online Database

## Change log: 
+ Fix typo in readme file
+ Add new version of StylePoint
+ Change script to Build instead of push
+ Delete default set of the development variable

## Additional changes:
+ Add new environment file to gitignore
+ Add the logic to set:
  + development = .env_development 
  + staging = .env_staging 
  + production = .env_production 
+ Set a general error for the environment variables
+ The method to start the application is: **set NODE_ENV=staging&&npm start**

**Note:**
The following env variables are needed for this code to work properly:
+ PORT=
+ SALT_FACTOR=
+ TOKEN_LIFE=
+ NODE_ENV=
+ EMAIL=
+ EMAIL_PASS=
+ SECRET=
+ DB_URL=
+ EMAIL_SERVICE=
+ PASSWORD_LENGHT=

### CODE REVIEW

**Comment by Ivan-Marquez** [comment]
We need the names of the required `.env` files in the PR description.

**Comment by kevinpr007** [comment]
Done!

File: package.json [hunk 1]
```diff
@@ -10,7 +10,7 @@
     "retire": " retire --node --verbose",
     "nodemon": "nodemon ./bin/www",
     "nsp": "nsp check",
-    "push": "npm run standard && npm run retire && npm run nsp"
+    "build": "npm run standard && npm run retire && npm run nsp"
```
Comment (Ivan-Marquez) [inline package.json]: This is more like a `validate` or `lint` script. This is optional, of course.

Comment (kevinpr007) [inline package.json]: We can this like that... to continue adding more commands

File: src/config/config.js [hunk 2]
```diff
@@ -1,12 +1,28 @@
...
+      path: `${__dirname}/.env_staging`
+    })
+    break
+  case 'production':
+    dotenv.load({
+      path: `${__dirname}/.env_production`
+    })
+    break
+  default:
+    throw new Error('Error setting the environmet variables.')
```
Comment (kevinpr007) [inline src/config/config.js]: Done!

**Pull request merged**